Gemfile

```diff
@@ -100,6 +100,7 @@ group :development, :test do
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
   gem 'dotenv-rails', '>= 2.7.5'
   gem 'erb_lint', require: false
+  gem 'htmlentities'
```

Comment:
This doesn't seem to be related to this PR. Has it crept in from another change?

Reply:
No - I included it because I was getting spec failures when Faker generates a name like "O'Keefe" - so this gem enables us to test against those names when they crop up as well.